examples/complete/main.tf

```diff
@@ -9,12 +9,14 @@ module "access_logs_bucket" {
 
 module "ecs_service_lb" {
   #checkov:skip=CKV_AWS_150: "Ensure that Load Balancer has deletion protection enabled"
+  #checkov:skip=CKV_AWS_290: "Ensure IAM policies does not allow write access without constraints"
```

Comment:
These exceptions are missing from the changelog

Reply:
Addressed @hugoalmeida264